### PR TITLE
Move to new Quark

### PR DESCRIPTION
**Pull request by superpoussin22**
### Describe the Pull Request

```
revamp the code to be able to use the new quark version

```

### Checklist for PR

- [X] Run MobSF unit tests and lint `tox -e lint,test`
- [X] Tested Working on Linux, Mac, Windows, and Docker
- [X] Add unit test for any new Web API (Refer: `StaticAnalyzer/tests.py`)
- [X] Make sure tests are passing on your PR [MobSF tests](https://github.com/MobSF/Mobile-Security-Framework-MobSF/actions)

### Additional Comments (if any)

```
DESCRIBE HERE
```

### CODE REVIEW

File: mobsf/MalwareAnalyzer/views/quark.py [hunk 1]
```diff
@@ -24,38 +24,23 @@ def quark_analysis(app_dir, apk_file):
...
-
-        json_report = data.get_json_report()
-
-        # Clear all functools.lru_cache from quark
-        data.apkinfo.find_method.cache_clear()
-        data.apkinfo.upperfunc.cache_clear()
-        data.apkinfo.get_wrapper_smali.cache_clear()
-        data.apkinfo.construct_bytecode_instruction.cache_clear()
+        # Generate Report
+        json_report = report.get_report('json')
```
Comment (haeter525) [inline mobsf/MalwareAnalyzer/views/quark.py:44]: The clear_cache() calls can explicitly release the resource.

```suggestion
        # Clear all functools.lru_cache from quark
        report.quark.apkinfo.find_method.cache_clear()
        report.quark.apkinfo.upperfunc.cache_clear()
        report.quark.apkinfo._construct_bytecode_instruction.cache_clear()
        report.quark.apkinfo.get_wrapper_smali.cache_clear()
```

**Comment by haeter525** [comment]
Hi, @superpoussin22 !

The test failed because Windows does not allow deleting files currently opened by a program.
Quark caches the targeted APK during the analysis. But, the APK was not automatically released after it finishes.

I suggest keeping the clear_cache() calls at lines 55~58. It explicitly releases the APK at the end of the analysis.

I have left a comment in the relevant code. Thank you :)

File: mobsf/MalwareAnalyzer/views/quark.py [hunk 2]
```diff
@@ -24,40 +24,29 @@ def quark_analysis(app_dir, apk_file):
...
     from quark import config
     from quark.freshquark import download
-    from quark.Objects.quark import Quark
-    from quark.Objects.quarkrule import QuarkRule
+    from quark.report import Report
 
     logger.info('Running Quark %s', quark_ver)
     json_report = {}
     try:
+        json_report = ''
```
Comment (ajinabraham) [inline mobsf/MalwareAnalyzer/views/quark.py:32]: should this be removed?

**Comment by superpoussin22** [comment]
@ajinabraham sounds good to me , if you can review :)

**Review by ajinabraham — DISMISSED** [review]
LGTM 👍  Tested.
Will keep quark disabled by default for sometime and if it is stable, we will revisit this later.

**Pull request merged**